### DISCUSSION TITLE
How to resize inline image when composing a message

### DISCUSSION THREAD

**Discussion by pepelton** [the-djmaze/snappymail#1043] [Q&A]
Hello,

I've updated my Rainloop webmail with Snappymail.

In Rainloop, when adding an inline image to an email, it is possible to set the width/height of the image.

I have not found a way to do this in Snappymail's editor (Squire). Is this possible, and if it is, how?

If resizing the image is not possible, would it be at least possible to set the default size so, that inline images fit into the compose window? 

Thank you,
Peter

**Comment by the-djmaze** [comment]
Please read #262 why Squire does not do this.
At the end i explain that SnappyMail now auto-resize images to max 1024px.

**Comment by pepelton** [comment]
Thank you for your reply.

Ok, the image size is not perfect on a big screen when reading (viewing) messages, but it is usable. 

But the image is too large in the compose window (editor). Here is what I did:

1. Took a screenshot from a big screen with Mac's screenshot built-in tool
2.  Started writing a new message and added the screenshot as an inline image
3. The image is shown way too big in the compose window (screenshot of the editor below)

Can anything be done here to make the image scale in a more reasonable way in the compose (editor) window?


<img width="999" alt="Screenshot 2023-03-30 at 11 25 09" src="https://user-images.githubusercontent.com/367221/228776007-05d08f48-948c-4696-8772-4f04e3b64a2e.png">

**Reply by the-djmaze** [reply]
> Can anything be done here to make the image scale in a more reasonable way in the compose (editor) window?

Not at the moment.

**Comment by the-djmaze** [comment]
I thought about it, maybe `max-width:100%` is sufficient.
It will then always scale to user screen size without side-scroll
